packages/amplication-client/src/Entity/EntityFieldListItem.scss

```diff
@@ -12,13 +12,24 @@
       margin-bottom: 0;
     }
   }
+  &__description-row {
+    padding-bottom: 12px;
+    border-bottom: 1px solid var(--black10);
```

Comment:
please use params instead of fixed values

Reply:
Replaced the `padding-bottom` value with a variable.